specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -0,0 +1,157 @@
+{
+    "swagger": "2.0",
+    "info": {
+      "version": "2018-11-01-preview",
+      "title": "BillingManagementClient",
+      "description": "Billing client provides access to billing resources for Azure subscriptions."
+    },
+    "host": "management.azure.com",
+    "schemes": [
+      "https"
+    ],
+    "consumes": [
+      "application/json"
+    ],
+    "produces": [
+      "application/json"
+    ],
+    "paths": {
+      "/providers/Microsoft.Subscription/subscriptions/{subscriptionId}/cancel": {
+        "post": {
+          "operationId": "CancelSubscription",
+          "description": "Cancels the subscription",
+          "parameters": [
+            {
+              "name": "subscriptionId",
+              "in": "path",
+              "description": "Subscription Id.",
+              "required": true,
+              "type": "string"
+            },
+            {
+              "$ref": "#/parameters/apiVersionParameter"
+            }
+          ],
+          "tags": [
+            "SubscriptionController"
+          ],
+          "responses": {
+            "200": {
+              "description": "OK. The request has succeeded.",
+              "schema": {
+                "$ref": "#/definitions/CancelSubscriptionResult"
+              }
+            },
+            "default": {
+              "description": "Error response describing why the operation failed.",
+              "schema": {
+                "$ref": "#/definitions/ErrorResponse"
+              }
+            }
+          }
+        }
+      },
+      "/providers/Microsoft.Subscription/subscriptions/{subscriptionId}/rename": {
+        "post": {
+          "operationId": "RenameSubscription",
+          "description": "Renames the subscription",
+          "parameters": [
+            {
+              "name": "subscriptionId",
+              "in": "path",
+              "description": "Subscription Id.",
+              "required": true,
+              "type": "string"
+            },
+            {
+              "$ref": "#/parameters/apiVersionParameter"
+            }
+          ],
+          "tags": [
+            "SubscriptionController"
+          ],
+          "responses": {
+            "200": {
+              "description": "OK. The request has succeeded.",
+              "schema": {
+                "$ref": "#/definitions/RenameSubscriptionResult"
+              }
+            },
+            "default": {
+              "description": "Error response describing why the operation failed.",
+              "schema": {
+                "$ref": "#/definitions/ErrorResponse"
+              }
+            }
+          }
+        }
+      }
+    },
+    "definitions": {
+      "CancelSubscriptionResult": {
+        "description": "Canceled Subscription Id",
+        "properties": {
+          "value": {
```

Comment:
>value [](start = 11, length = 5)

Can we make the response more specific? Perhaps rename the property to canceledSubscriptionId or something like:
{
  "canceledSubscription": { ... entire payload of cancelled subscription ... }
}

Reply:
Made the changes